coralproject/elkhorn
# Publish gallery

## What?

this is the pair PR for https://github.com/coralproject/cay/pull/525

please check out both branches to test.
## How to test
- go through the items in https://github.com/coralproject/cay/pull/525
- ALSO, make sure that I didn't break anything related to publishing a form, since I had to move some stuff around
